scripts/configure.sh

```diff
@@ -315,6 +315,13 @@ if [ $native_build == 1 ] ; then
     }
     source ~/.bashrc
 
+    if [[ -z "${ZSH_VERSION}" ]]; then
+        echo "Setting .zshrc with environment variables..."
+        echo '# Astrobee environment variables' >> ~/.zshrc
+        echo 'source /opt/ros/'$ros_version'/setup.zsh' >> ~/.zshrc
+        echo 'if [[ ":$CMAKE_PREFIX_PATH:" != *":'${cmake_astrobee_path}':"* ]]; then CMAKE_PREFIX_PATH="${CMAKE_PREFIX_PATH:+"$CMAKE_PREFIX_PATH:"}'${cmake_astrobee_path}'"; fi' >> ~/.zshrc
```

Comment:
I second Oleg's comment, if you notice we have before those add commands:
 grep -qF ${cmake_astrobee_path} ~/.bashrc ||